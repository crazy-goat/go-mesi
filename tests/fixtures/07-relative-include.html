<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
This should print hello world: [<esi:include src="/hello" />]
This should print hello world: [<esi:include src="hello" />]
</body>
</html>
