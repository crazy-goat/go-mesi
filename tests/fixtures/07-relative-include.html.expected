<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
This should print hello world: [Hello World]
</body>
</html>
